.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,83 @@
+name: Docker Smoke Test
+
+on:
+  push:
+    branches: [master, develop]
+  pull_request:
+    branches: [master, develop]
+
+env:
+  DHIS2_VERSION: ${{ vars.DHIS2_VERSION }}
+  POSTGRESQL_VERSION: ${{ vars.POSTGRESQL_VERSION }}
+  HEALTH_CHECK_ENDPOINT: ${{ vars.HEALTH_CHECK_ENDPOINT }}
+  POSTGRESQL_USERNAME: ${{ secrets.POSTGRESQL_USERNAME }}
+  POSTGRESQL_PASSWORD: ${{ secrets.POSTGRESQL_PASSWORD }}
+  POSTGRESQL_DATABASE: ${{ secrets.POSTGRESQL_DATABASE }}
+  POSTGRESQL_POSTGRES_PASSWORD: ${{ secrets.POSTGRESQL_POSTGRES_PASSWORD }}
+
+jobs:
+  smoke-test:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Start services
+        run: |
+          docker compose up -d
+          docker compose ps
```

Comment:
The `ps` command seems redundant here. If there are issues or anything we have the debug step at the bottom.